src/main/java/jenkins/plugins/git/OpenSSHKeyFormatImpl.java

```diff
@@ -0,0 +1,106 @@
+package jenkins.plugins.git;
+
+import hudson.FilePath;
+import io.jenkins.cli.shaded.org.apache.sshd.common.config.keys.loader.openssh.OpenSSHKeyPairResourceParser;
+import io.jenkins.cli.shaded.org.apache.sshd.common.NamedResource;
+import io.jenkins.cli.shaded.org.apache.sshd.common.config.keys.FilePasswordProvider;
+import io.jenkins.cli.shaded.org.apache.sshd.common.config.keys.writer.openssh.OpenSSHKeyPairResourceWriter;
+import io.jenkins.cli.shaded.org.apache.sshd.common.session.SessionContext;
+import io.jenkins.cli.shaded.org.apache.sshd.common.util.io.SecureByteArrayOutputStream;
```

Comment:
With the other two changes (here and here) that precede this change, I was able to switch to directly import the sshd-common symbols.
```suggestion
import org.apache.sshd.common.config.keys.loader.openssh.OpenSSHKeyPairResourceParser;
import org.apache.sshd.common.NamedResource;
import org.apache.sshd.common.config.keys.FilePasswordProvider;
import org.apache.sshd.common.config.keys.writer.openssh.OpenSSHKeyPairResourceWriter;
import org.apache.sshd.common.session.SessionContext;
import org.apache.sshd.common.util.io.SecureByteArrayOutputStream;
```